### PR TITLE
[SPEC] Add support string as AutoBroadcastSpec

### PR DESCRIPTION
**Pull request by mbencer**
According to specification `AutoBroadcastSpec` should be passed as string. It's my proposition to support this case.

### CODE REVIEW

File: src/ngraph/op/util/attr_types.cpp [hunk 1]
```diff
@@ -126,4 +128,17 @@ namespace ngraph
     {
         return s << as_string(type);
     }
+
+    op::AutoBroadcastType op::AutoBroadcastSpec::type_from_string(const std::string& type) const
+    {
+        static const std::map<std::string, AutoBroadcastType> allowed_values = {
+            {"none", AutoBroadcastType::NONE},
```
Comment (diyessi) [inline src/ngraph/op/util/attr_types.cpp]: Let's keep the strings and enum values consistent.

Comment (mbencer) [reply src/ngraph/op/util/attr_types.cpp]: Done

File: src/ngraph/op/util/attr_types.hpp [hunk 2]
```diff
@@ -257,6 +257,10 @@ namespace ngraph
                 , m_axis(0)
             {
             }
+            AutoBroadcastSpec(const char* type)
```
Comment (tsocha) [inline src/ngraph/op/util/attr_types.hpp]: Could we use `std::string` here?

Comment (mbencer) [reply src/ngraph/op/util/attr_types.hpp]: I consider it but `std::string` can not be used to implicit conversion here. It doesn't compile.

**Pull request merged**